content/docs/reference/cli-connection-string.md

```diff
@@ -63,6 +63,12 @@ In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-opt
    postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname?connect_timeout=30
     ```
 
+- Get a connection string to a specific point in a branch's history by appending `@timestamp` or `@lsn`. Availability depends on your configured [history retention](/docs/manage/projects#configure-history-retention) window.
+
+    ```bash
+    neonctl connection-string @2024-04-21T00:00:00Z
+    ```
+
```

Comment:
For additional examples, see [How to use Time Travel](/docs/guides/time-travel-assist#how-to-use-time-travel).